### PR TITLE
fix: slack icon to discord icon in dropdown

### PR DESCRIPTION
**Pull request by akxcix**
## What does this PR do?

there was a dropdown menu item with discord's link but slack's icon. this pr aims to change that to discord's icon.

existing light modeexisting dark mode

Fixes # (issue)
fixed light modefixed dark mode

## Requirement/Documentation

  - If there is a requirement document, please, share it here.
  - If there is ab UI/UX design document, please, share it here.

## Type of change

  - [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- http://localhost:3000/event-types
- click on the menu on the top left
- in the dropdown menu the item is at the 4th position

## Mandatory Tasks

  - [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by akxcix** [comment]
some tests were failing locally, will figure out after work

**Review by PeerRich — DISMISSED** [review]

File: packages/features/shell/Shell.tsx [hunk 1]
```diff
@@ -73,12 +73,31 @@ import {
...
+        </svg>
+    </div>
+  )
+}
+
```
Comment (PeerRich) [inline packages/features/shell/Shell.tsx:100]: i wonder if we should make this an export, and reuse on our 404 page, too: http://cal.com/404-page

**Review by PeerRich — APPROVED** [review]

**Comment by PeerRich** [comment]
better solution: https://github.com/calcom/cal.com/pull/10193/files

**Comment by akxcix** [comment]
agreed, gravitated towards that in the end. thanks for taking the time to review though!

**Pull request closed without merging**